app/services/github_service.rb

```diff
@@ -1,15 +1,13 @@
-# frozen_string_literal: true
-
 class GithubService
   def repo_info
-    params = { part: 'owner', id: id }
-    JSON.parse(conn.get('/user/repos', params).body, symbolize_names: true)
+    JSON.parse(conn.get('/user/repos').body, symbolize_names: true)
```

Comment:
Why did you take out the params?

Reply:
We weren't actually passing any params in so the variable wasn't doing anything